control/subscription_group.go

```diff
@@ -51,13 +51,12 @@ type ManagesSubscriptionGroups interface {
 	Add(id string, requested []core.RequestedMetric,
 		configTree *cdata.ConfigDataTree,
 		plugins []core.SubscribedPlugin) []serror.SnapError
-	Get(id string) ([]core.Metric, []serror.SnapError, error)
+	Get(id string) (map[string]metricTypes, []serror.SnapError, error)
```

Comment:
It seems out of place for this to not take a type from core and instead uses `metricTypes` which is different than `metricType`.